### PR TITLE
Write basic documentation for koji builder

### PR DESCRIPTION
**Pull request by khardix**
Requested in sclorg#76.

I did attempt to stick to the style of the rest of the user guide,
while describing all relevant CLI options.

Let me know if you would like me to add or change anything.

### CODE REVIEW

**Comment by khardix** [comment]
Test fails on stylistic check not related to the changes in this PR.

**Comment by junaruga** [comment]
> Test fails on stylistic check not related to the changes in this PR.

The reason of the style check is because `flake8` latest version 3.5.0 added new check.
The issue is not related to this PR.

I fixed it in an another project.
https://github.com/junaruga/rpm-py-installer/pull/93/files

File: docs/users_guide.md [hunk 1]
```diff
@@ -243,6 +243,40 @@ For both hooks, the environment variable `CUSTOM_DIR` refers to the directory co
...
+          --build koji \
+          --koji-epel EPEL_VERSION \
+          --koji-profile PROFILE \
+          --koji-owner OWNER \
+          ...
+          RECIPE_FILE \
+          COLLECTION_ID
+
+    Here, `EPEL_VERSION` is the major EL version you wish to build for
+    (i.e. `7` for CentoOS 7), `PROFILE` names the configuration profile
```
Comment (junaruga) [inline docs/users_guide.md]: Typo: CentoOS -> CentOS

File: docs/users_guide.md [hunk 2]
```diff
@@ -243,6 +243,40 @@ For both hooks, the environment variable `CUSTOM_DIR` refers to the directory co
           RECIPE_FILE \
           COLLECTION_ID
 
+#### Koji build
+
+1.  Ensure that you have configuration for the desired koji instance
+    (i.e.  Fedora Koji, CentOS CBS, …) installed on your system.
```
Comment (junaruga) [inline docs/users_guide.md]: I want links for
Fedora Koji to https://koji.fedoraproject.org/koji/
CentOS CBS to https://cbs.centos.org/koji/

How?

**Comment by junaruga** [comment]
Thanks! Merged with squash.

**Pull request merged**